manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/builder/DefaultPortsBuilder.java

```diff
@@ -46,6 +46,7 @@
 public class DefaultPortsBuilder {
 
     private static final Logger logger = LoggerFactory.getLogger(DefaultPortsBuilder.class);
+    private static String seperator = "/";
```

Comment:
Let's move this to ToolConstants as public static final String PORTS_PROTOCOL_SEPARATOR = "/". And use that everywhere